apps/finance/app/src/components/TransfersFilters.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { DropDown, springs, theme, unselectable } from '@aragon/ui'
+import { Spring, animated } from 'react-spring'
+import DownloadButton from './DownloadButton'
+import DateRange from './DateRange/DateRangeInput'
+
+const TransfersFilters = ({
```

Comment:
Small nitpick: what do you think about `TransferFilters` instead as the name?

Reply:
I actually started by having it named `TransferFilters`, then I renamed it to `TransfersFilters`, not only because it applies to transfers (it’s ok to not be strictly correct if a name is nicer), but because it is a component that is meant to be used from the `Transfers` component. What do you think?

Reply:
Makes sense 👍